src/WP_Export_Stdout_Writer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class WP_Export_Stdout_Writer extends WP_Export_Base_Writer {
+
+	function __construct( $formatter, $writer_args = array() ) {
+		parent::__construct( $formatter );
+		//TODO: check if args are not missing
+		$this->before_posts_xml = $this->formatter->before_posts();
+		$this->after_posts_xml = $this->formatter->after_posts();
+	}
+
+	public function export() {
+		// WP_CLI\Utils\wp_clear_object_cache(); ?
```

Comment:
Is there a reason this is commented out?

Reply:
dunno